cms/templates/index.html

```diff
@@ -443,10 +443,25 @@ <h3 class="course-title">${library_info['display_name']}</h3>
       <div class="notice notice-incontext notice-instruction notice-instruction-nocourses list-notices libraries-tab">
         <div class="notice-item">
           <div class="msg">
```

Comment:
Yes, there is a single "creator" privilege that affects the ability to create courses and libraries.

If you look above in this file, you can see the button in the top right checks for `if course_creator_status=='granted':` before showing the "Create Library" button. The same check should be added to the new button.